### ISSUE TITLE
Compilation very slow

### ISSUE THREAD

**Issue by animator013** [craigburke/typescript-asset-pipeline#6]
I hope you'll look into this.

There is a big performance bottleneck when using java javascript engine. I also tried the nashorn for java 8, which improved performance but it is still slow.

I just bootstrapped some angular2 project with primeng library and compilation took about 7s. Which is very hard for development.

Thank you for your time and effort you put into this.

**Comment by craigburke** [comment]
@animator013 yeah it is definitely still a proof of concept at this point. I hope to be able to set aside some time soon to take it a bit further than that.
